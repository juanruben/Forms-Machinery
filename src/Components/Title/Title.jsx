import React from 'react';
import PropTypes from 'prop-types';
import './Title.scss';

function Title(props) {
    const { text } = props;
    return (
        <div className="main-title">
            {text}
        </div>
    );
}

Title.propTypes = {
    text: PropTypes.string.isRequired,
};

export default Title;
