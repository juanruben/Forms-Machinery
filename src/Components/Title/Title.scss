@import "../../styles.scss";

.main-title {
  border-bottom: solid 1px #ddd;
  font-size: 1.5em;
  font-weight: bold;
  padding-bottom: 5px;
}
