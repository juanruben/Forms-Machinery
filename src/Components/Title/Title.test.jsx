import React from 'react';
import { shallow } from 'enzyme';
import Title from './Title';

describe('Title', () => {
    let wrapper;

    it('should render like snapshot', () => {
        wrapper = shallow(<Title text="Test" />);
        expect(wrapper.debug()).toMatchSnapshot();
    });
});
