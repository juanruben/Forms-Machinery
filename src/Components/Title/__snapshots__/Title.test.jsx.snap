// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should render like snapshot 1`] = `
"<div className=\\"title-container\\">
  Test
</div>"
`;
